orttraining/orttraining/test/python/orttraining_ortmodule_tests.py

```diff
@@ -38,6 +38,15 @@ def run_ortmodule_api_tests(cwd, log, transformers_cache):
     run_subprocess(command, cwd=cwd, log=log, env=env).check_returncode()
 
 
+def run_ortmodule_fallback_tests(cwd, log, transformers_cache):
```

Comment:
Seems like these tests are not enabled? We need to add a line in ```main```.

Reply:
weird, must have been lost in one of the conflicts. Thanks for catching this up!